src/ed25519pod.rs

```diff
@@ -399,12 +422,13 @@ fn pub_self_statements(msg: &[u8], pk: &[u8]) -> Vec<middleware::Statement> {
 pub mod tests {
     use std::any::Any;
 
-    use pod2::{self, frontend::MainPodBuilder, op};
+    use pod2::{self, frontend::MainPodBuilder, middleware::VDSet, op};
     use ssh_key::SshSig;
 
     use super::*;
 
     #[test]
+    #[ignore] // This is for the GitHub CI, it takes too long and the CI would fail.
```

Comment:
Note: this `ignore` is as @ainta initially had it in PR #2 , I suggested removing it to enforce this test too, but it seems that the GitHub CI fails because it takes too long. Disabling it again, and for now we can double check running the tests locally at new PRs, but eventually we can find a solution (either increasing GitHub's cuota, or by setting our own custom server for the CI).